tools/PowerShell/Microsoft.WinGet.Client/src/Library/Enable-WinGetLocalManifest.ps1

```diff
@@ -0,0 +1,41 @@
+Function Enable-WinGetLocalManifest
```

Comment:
The PowerShell cmdlet structure tends to use Enable and Disable. It makes them more discoverable. Maybe someone from the community can weigh in on this one. @doctordns what do you think?

Reply:
I just expect the possibility of non-boolean values in the future, but it could also be `Enable`/`Disable` for booleans and `Set` in the future for non-booleans.  But the larger takeaway is to make the actual setting that is being modified be a parameter rather than making new cmdlets per setting.

Reply:
Enable, in general moves a resource from a disabled to enabled state - Disable doing the opposite. I think, in general, I'd prefer Set - one command to set/unset vs two cmdlets (enable and disable).  John is right - there could be some resources that need to be set to a non boolean value.